### DISCUSSION TITLE
Add 9th gamepad input

### DISCUSSION THREAD

**Discussion by tautologias345** [godotengine/godot-proposals#5694] [3D]
A 1920x1080 full screen that is a split screen with 3 rows and 3 columns, 640x360 each is well seen, as on desktop as on mobile. So, how can I play on 9 players with gamepad?

**Comment by Calinou** [comment]
Does any operating system even support having more than 8 controllers connected at a given time? I remember reading that most modern APIs are limited to 4 controllers in fact (including latest-generation consoles).
